components/DropdownLight/index.d.ts

```diff
@@ -0,0 +1,27 @@
+
+// Generated with scripts/create-component.js
+import { Component } from 'react';
+
+export type ItemPropType = string | { value: string | number; label: string };
+
+
+export interface DropdownLightProps {
+  disabled?: boolean;
+  label?: string;
+  placeholder?: string;
+  selectedItem?: ItemPropType;
+
+  onChange?: (
+    selectedItem: ItemPropType | null,
+  ) => void;
+  items?: ItemPropType[];
```

Comment:
When placing the items property as required, it is necessary to adjust this item by removing the character "?"

Reply:
I didn't understand

Reply:
The character "?" on a typescript property indicates that it is optional. Since the "items" property must be required, this character is not needed here

Reply:
you are right i removed it from props but not from suggestions